### ISSUE TITLE
Language support for 403,503,404

### ISSUE THREAD

**Issue by Blackswirllife** [bolt/bolt#7966]
Hey :)

Would it be possible to get language support for the 403 503 404 pages as well?

If I create a 404 page under Blocks and change the language it stays on the default language.

I have already tried in the documentation to find what about it, unfortunately in vain.

**Comment by bobdenotter** [comment]
Dumb question: If you get a `404`, it's (by definition) a page that doesn't exist. How would you determine what locale to use to show to the user?

In my opinion, the `404` page should have a message in the default locale, plus inlined in the same page, a short message in other supported locales.

**Comment by Blackswirllife** [comment]
That's true, but that only applies to a static page in my opinion. We have wrapped the error page in the _master main and when the 404 is displayed, the language of the menu changes back to German, although the user has set the page to English.

When we put a link in to get back to the home page, this user comes back from English to the German home page, which affects the user friendliness a bit.

**Comment by Halopend** [comment]
So my trick for the 404 page has been to use notation like this

{{‘Whoops! This page does seem to exist’|trans}}

To allow the xlf translation engine to be used (basically, a more application/development focused translation engine built into symfony you can access either directly through a file or the symfony debugger).

Edit: I should add the documentation mentions using blocks but I could not get the content entry side of bolt to talk to the 404 (I’d get an error) but it was more so related to my master.twig having references to record which doesn’t apply to the 404 page).
